python_bindings/cpp_bindings/bindings.cpp

```diff
@@ -952,9 +952,9 @@ class BFIndex
     }
 };
 
-PYBIND11_PLUGIN(hnswlib)
+PYBIND11_PLUGIN(cpp_bindings)
```

Comment:
does this change the name of the module / what does this change do?

Reply:
this changes the name of the python bindings package